README.md

```diff
@@ -1,4 +1,5 @@
 # Spryker B2C Demo Shop
+[![Build Status](https://travis-ci.com/spryker/b2c-demo-shop-internal.svg?token=uuNLarTLvJGpcdGUgqj2&branch=master)](https://travis-ci.com/spryker/b2c-demo-shop-internal)
```

Comment:
these links are invalid (wrong repo)

Reply:
Please make sure that both b2c and b2b public ones have the right fix for URLs when you merge over from internal one.